frontend/app/routes/public/renew/$id/applicant-information.tsx

```diff
@@ -156,6 +172,7 @@ export default function RenewApplicationInformation() {
 
   return (
     <>
+      {/* Todo: add conditional check to show <StatusNotFound> if clientApplication returned null */}
```

Comment:
Use `TODO`, not `Todo`.. IDEs look for the former to create markers in the code so you can easily find them.

ex:
![image](https://github.com/user-attachments/assets/668e6b8c-a9db-43af-8d5f-5450fdca89e9)